infra/ansible/roles/frontend/tasks/main.yml

```diff
@@ -45,6 +45,8 @@
   notify:
     - Install Node
     - Yarn install
+    - Check assetlinks.json
+    - Copy assetlinks.json
```

Comment:
why is this necessary? the content of public/.well-known should be served directly with the rest lf the public/ content.
Just like https://tournesol.app/.well-known/security.txt no?

Reply:
ah ok I’ve just realized the file committed in .well-known contains the domain name in the filename. So this is an internal convention?

Reply:
Yes, the goal is to commit all files `assetlinks.json` for each domain, staging, production, and let Ansible define the correct file during the deployment.